cli/src/main/resources/help/shutdown.adoc

```diff
@@ -26,11 +26,12 @@ Stops the server to which the CLI is connected.
 Stops the server with hostname `my_server01`.
 
 `shutdown cluster` +
-Stores cluster state, persists entries if you use a cache store, and stops all nodes.
+Stops all nodes in the cluster after storing cluster state and persisting entries if there is a cache store.
 
 `shutdown container` +
-Stores cluster state and persists entries if you use a cache store. The servers remain running with active endpoints
-and clustering, however REST calls to container resources will result in a 503 Service Unavailable response.
+Stops the data container without terminating the server process. Stores cluster state and persists entries if there is a cache store. Server instances remain running with active endpoints and clustering. REST calls to container resources will result in a 503 Service Unavailable response.
++
+The `shutdown container` command is not intended for use with cluster administration. If you stop the data container you should use the `shutdown cluster` command to stop the cluster before restarting it.
```

Comment:
This isn't right. Executing `shutdown cluster` after `shutdown container` will result in a 503 as it's trying to perform actions on the container resource which we have just shutdown. Also, the purpose of the command is for cluster administration, albeit it probably won't apply to most users.

I think we should just state something like: "The `shutdown container` command is intended for environments, such as Kubernetes, that manage resource lifecycles independently. Unless you explicitly want server processes to remain, without an active container, then it's likely that you should use the `shutdown cluster` command instead."

Reply:
I think you've nailed it. Cheers @ryanemerson